cmd/skopeo/utils.go

```diff
@@ -35,7 +35,8 @@ func commandAction(handler func(args []string, stdout io.Writer) error) cli.Acti
 // sharedImageOptions collects CLI flags which are image-related, but do not change across images.
 // This really should be a part of globalOptions, but that would break existing users of (skopeo copy --authfile=).
 type sharedImageOptions struct {
-	authFilePath string // Path to a */containers/auth.json
+	authFilePath         string // Path to a */containers/auth.json
+	bigFilesTemporaryDir string // A directory to use for big temporary files
```

Comment:
Please use `globalOptions`, and a flag in `createApp()`, for this; `sharedImageOptions` is mostly a hack for `--authfile`.